.eslintrc.js

```diff
@@ -1,18 +1,26 @@
 module.exports = {
   extends: ['react-app', 'prettier'],
+  extends: [
+    'eslint:recommended',
```

Comment:
Adding `'eslint:recommended'` is IME generally a good idea for all JS/TS codebases because it has a lot of good built-in rules. Was it intentional not to have it previously?